RELEASE_NOTES/2.0.0-internal.8.0.0.md

```diff
@@ -0,0 +1,259 @@
+
+This interface is deprecated and was not intended for public export.
+
+## merge-tree: Remove several APIs
+
+Removed the following APIs:
+
+- Stack
+- clone
+- combine
+- createMap
+- extend
+- extendIfUndefined
+- matchProperties
+
+This functionality is deprecated and was not intended for public export.
+
+- `AzureAudience` (use `IAzureAudience` instead)
+- `TinyliciousAudience` (use `ITinyliciousAudience` instead)
+- `DOProviderContainerRuntimeFactory`
+- `FluidContainer`
+- `ServiceAudience`
+
+## runtime-definitions: Removed request and IFluidRouter from IDataStore
+
+The `request` method and `IFluidRouter` property have been removed from `IDataStore`. Please migrate all usage to the `IDataStore.entryPoint` API.
+
+See [Removing-IFluidRouter.md](https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-interfaces/Removing-IFluidRouter.md) for more details.
+
+## runtime-definitions: Removed IFluidRouter from IFluidDataStoreChannel and FluidDataStoreRuntime
+
+The `IFluidRouter` property has been removed from `IFluidDataStoreChannel` and `FluidDataStoreRuntime`. Please migrate all usage to the `IFluidDataStoreChannel.entryPoint` API.
+
+See [Removing-IFluidRouter.md](https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-interfaces/Removing-IFluidRouter.md) for more details.
+
+## runtime-utils: Removed requestFluidObject
+
+The `requestFluidObject` utility has been removed. Please migrate all usage of it to the new `entryPoint` pattern.
+
+See [Removing-IFluidRouter.md](https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-interfaces/Removing-IFluidRouter.md) for more details.
+
+## sequence: Some function return types are now void instead of any
+
+The return types of some functions have changed from `any` to `void` because the projects are now being compiled with the `noImplicitAny` TypeScript compilation option. This does not represent a logic change and only serves to make the typing of these functions more accurate.
+
+## sequence: Add experimental support for the obliterate operation
+
+This change adds experimental support for *obliterate*, a form of *remove* that deletes concurrently inserted segments. To use, enable the `mergeTreeEnableObliterate` feature flag and call the new `obliterateRange` functions.
+
+Note: this change may cause compilation errors for those attaching event listeners. As long as obliterate isn't used in current handlers, their current implementation is sound.
+
+## sequence: Removed Marker.hasSimpleType and made sequence operations return void
+
+`Marker.hasSimpleType` was unused. Sequence operations now no longer return IMergeTree\*Msg types. These types are redundant with the input.
+
+## sequence: Removed several public exports from merge-tree and sequence
+
+The following APIs have been removed or marked internal in merge-tree and sequence. This functionality was never intended for public export.
+
+- BaseSegment.ack
```

Comment:
About half the items listing removed APIs in this file are quoted as code, and the other half arn't. Would be nice to be consistent.

Reply:
Should be fixed in latest. It's manual cleanup so if you notice anything else let me know.